ambuild2/frontend/base/gen.py

```diff
@@ -199,16 +199,15 @@ def compileScript(self, path):
       chars = fp.read()
 
       # Python 2.6 can't compile() with Windows line endings?!?!!?
-      chars = chars.replace('\r\n', '\n')
-      chars = chars.replace('\r', '\n')
+      chars = chars.replace('\r\n', '\n').replace('\r', '\n')
```

Comment:
The previous style is intentional - the sequence of operations is clearer before.